src/plugins/intel_cpu/src/transformations/transformation_pipeline.cpp

```diff
@@ -470,6 +471,36 @@ void Transformations::PreLpt(const std::vector<ov::element::Type>& defaultPrecis
         },
         ov::pass::NormalizeL2Decomposition);
 
+    if (!useLpt) {
+        CPU_SET_CALLBACK_X64(manager,
+            [this](const_node_ptr &node) -> bool {
+                // This is a callback from snippets. If GroupNorm node is appropriate for snippets execution with higher perf,
+                // then it will not be decomposed to mvn+reshape+eltwises, support it with snippets instead.
```

Comment:
Please create a ticket with a description of current snippets limitations. Why do we have to fall back to the plugin pipeline in some cases. What should we do in snippets to be able to always execute GN better (or at least no worse) than the plugin?
Also, add the ticket reference to the comment.

Reply:
Not fully enabled because perf reason. CVS-124241 is describing it and updated to code comments. There are currently 2 sub tickets, extract_invariance pass and buffer_output inplace. I see at at least another two: To enale int8 and bf16, need fast pass reduce calculation. Move parallel to snippets for flexible scheduling. Will created new or update the ticket link. Thanks Ivan!